### PR TITLE
Reset-DbaAdmin - Use Resolve-DbaNetworkName to get ipaddress and hostname

### LINKED ISSUE #7529: [Bug] Reset-DbaAdmin - On Remote Execs Could Not Resolve IP And AddressList.IPAddressToString
### Report

Remote executions of this cmdlet fail with below.
`WARNING: [23:50:27][Reset-DbaAdmin] Could not resolve SqlServer IP or NetBIOS name | The specified operator requires both the -Property and -Value parameters. Provide values for both
parameters, and then try the command again.` [I think this issue was introduced on Line 197 made within this commit. ](https://github.com/sqlcollaborative/dbatools/commit/a1c053f8a1d1cf9a16a14909f015430cc75415c4#diff-eed4f75ea03b159e4ff89eeaf5f2b1da83e01ef580174d66f81689ebd175a22aR197-R200) I think the issue was the removal of the `$_` within the `Where-Object` block.

Additionally, on line [196](https://github.com/sqlcollaborative/dbatools/blob/b52e18d5f25d932b1d785589a6ffa0b4fd402541/functions/Reset-DbaAdmin.ps1#L196), where `$hostentry` is defined, the method call to `GetHostEntry` returns `IPHostEntry` [class](https://docs.microsoft.com/en-us/dotnet/api/system.net.iphostentry?view=net-5.0) on _my_ machine which doesn't immediately include the `IPAddressToString`. I had to retrieve it from `AddressList`.

#### Workaround
`Install-Module dbatools` locally and run `Reset-DbaAdmin -SqlInstance localhost -Login 'Domain\User' -WhatIf`.

#### Host used

- [x] powershell.exe
- [ ] ISE
- [ ] VS Code
- [ ] Other (please specify)

#### Errors Received

```
writeErrorStream      : True
PSMessageDetails      :
Exception             : System.Exception: The specified operator requires both the -Property and -Value 
parameters. Provide values for both parameters, and then try the command again.
                        ---> System.Management.Automation.PSArgumentException: The specified operator requires 
both the -Property and -Value parameters. Provide values for both
                        parameters, and then try the command again.
                           at System.Management.Automation.MshCommandRuntime.ThrowTerminatingError(ErrorRecord 
                           errorRecord)
                           --- End of inner exception stack trace ---
TargetObject          :
CategoryInfo          : InvalidArgument: (:) [Write-Error], Exception
FullyQualifiedErrorId : dbatools_Reset-DbaAdmin,Stop-Function
ErrorDetails          : The specified operator requires both the -Property and -Value parameters. 
Provide values for both parameters, and then try the command again.
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at Stop-Function, 
C:\Program Files\WindowsPowerShell\Modules\dbatools\1.0.170\allcommands.ps1: line 89813
                        at Reset-DbaAdmin<Process>, 
C:\Program Files\WindowsPowerShell\Modules\dbatools\1.0.170\allcommands.ps1: line 64807
                        at <ScriptBlock>, <No file>: line 1
PipelineIterationInfo : {0, 1}
```

#### Steps to Reproduce

Run `Reset-DbaAdmin -SqlInstance $RemoteServer -Login 'Domain\User' -WhatIf`

#### Expected Behavior

Add Domain\User as `sa` to $RemoteServer

#### Actual Behavior

Failure happens.

### Environmental information

🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
Please provide the output of the below script

```powershell
PowerShell Version        : 5.1.18362.1171
dbatools latest installed : 1.0.170
Culture of OS             : en-US
```

#### SQL Server:

```sql
Microsoft SQL Server 2017 (RTM) - 14.0.1000.169 (X64) 
	Aug 22 2017 17:04:49 
	Copyright (C) 2017 Microsoft Corporation
	Express Edition (64-bit) on Windows Server 2016 Standard 10.0 <X64> (Build 14393: ) (Hypervisor)
```

```sql
us_english
```

### PR DESCRIPTION
**Pull request by andreasjordan**
## Type of Change

 - [x] Bug fix (non-breaking change, fixes #7529 )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1`)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Unit test is included
 - [ ] Documentation
 - [ ] Build system

I hope this works, please test.
Not 100% sure if we need FullComputerName here.

### CODE REVIEW

File: functions/Reset-DbaAdmin.ps1 [hunk 1]
```diff
@@ -165,50 +165,10 @@ function Reset-DbaAdmin {
...
-                    # this is required otherwise, the ip is returned
-                    $hostName = (Get-WmiObject -Class Win32_NetworkAdapterConfiguration -Filter IPEnabled=TRUE -ComputerName $ipaddr -ErrorAction Stop).PSComputerName
-                    if ($null -eq $hostName) {
-                        $hostName = (nbtstat -A $ipaddr | Where-Object { $_ -match '\<00\>  UNIQUE' } | ForEach-Object { $_.SubString(4, 14) }).Trim()
-                    }
-                } catch {
-                    Stop-Function -Continue -ErrorRecord $_ -Message "Could not access remote WMI object. Check permissions and firewall."
-                }
+            } else {
+                $resolved = Resolve-DbaNetworkName -ComputerName $baseaddress
```
Comment (potatoqualitee) [inline functions/Reset-DbaAdmin.ps1:169]: Should you pass the $Credential here, too?

Comment (andreasjordan) [reply functions/Reset-DbaAdmin.ps1:169]: We should, but we don't have it. Should we add -Credential to the command?

Comment (potatoqualitee) [reply functions/Reset-DbaAdmin.ps1:169]: Yeah, the command supports it. Then we should probably create an issue if we missed passing it in other commands too

Comment (andreasjordan) [reply functions/Reset-DbaAdmin.ps1:169]: To be clear: Resolve-DbaNetworkName has a parameter Credential, but Reset-DbaAdmin has no parameter Credential. Should I add the parameter to Reset-DbaAdmin?

Comment (potatoqualitee) [reply functions/Reset-DbaAdmin.ps1:169]: ahhhh! good call. Yes, let me merge this then that can be a new PR 👍🏼

**Pull request merged**